uiowa/uiowa
# Find Text - Expanded access

Resolves #6974

# To Test

- Sync any site. Masquerade as a publisher and an editor and see that you can access the Find Text tool and use it.
